scene/gui/color_picker.h

```diff
@@ -248,7 +249,9 @@ class ColorPicker : public VBoxContainer {
 	void create_slider(GridContainer *gc, int idx);
 	void _reset_sliders_theme();
 	void _html_submitted(const String &p_html);
-	void _value_changed(double);
+	void _drag_started();
+	void _value_changed();
+	void _drag_ended();
```

Comment:
```suggestion
	void _slider_drag_started();
	void _value_changed();
	void _slider_drag_ended();
```
Makes it more clear.

Reply:
I'm not 100% sure about `_value_changed` change. I feel like `_slider_value_changed` would hide the fact that it is also called when SpinBox is used. Maybe `_slider_or_spin_value_changed`?

Reply:
In ColorPicker, SpinBoxes are treated as part of Sliders, so the name would be correct as it refers to the picker's "slider".

Reply:
😲😲😲